### PR TITLE
Fix the root configuration in the user interface

### PR DESCRIPTION
**Pull request by poncovka**
Ported from https://github.com/rhinstaller/anaconda/pull/3775.

### CODE REVIEW

File: pyanaconda/ui/gui/spokes/root_password.py [hunk 1]
```diff
@@ -179,24 +179,26 @@ def mandatory(self):
...
+            # Unlock the root account.
+            self._users_module.SetRootAccountLocked(False)
 
-        if enabled:
+        else:
+            # Reset the root password.
+            self._users_module.ClearRootPassword()
+
+            # Lock the root account.
+            self._users_module.SetRootAccountLocked(True)
```
Comment (jkonecny12) [inline pyanaconda/ui/gui/spokes/root_password.py:194]: Do we want to Lock the root if the root account is enabled but password is not set? I'm just not sure if that is the correct behavior :).

**Review by jkonecny12 — APPROVED** [review]
Looks good to me otherwise.

**Review by VladimirSlavik — APPROVED** [review]
Thank you!

File: pyanaconda/ui/gui/spokes/root_password.py (hunk 1 above)
Comment (VladimirSlavik) [reply pyanaconda/ui/gui/spokes/root_password.py:194]: I agree that porting it 1:1 is correct.

Comment (jkonecny12) [reply pyanaconda/ui/gui/spokes/root_password.py:194]: Thanks for explanation.

**Pull request merged**